core/src/wallet/cosmos/CosmosLikeAccount.cpp

```diff
@@ -595,25 +595,19 @@ namespace ledger {
 
                 std::shared_ptr<api::CosmosLikeTransactionBuilder> CosmosLikeAccount::buildTransaction(const std::string &senderAddress) {
                         auto self = std::dynamic_pointer_cast<CosmosLikeAccount>(shared_from_this());
-                        auto buildFunction = [self, senderAddress](const CosmosLikeTransactionBuildRequest &request,
-                                                                   const std::shared_ptr<CosmosLikeBlockchainExplorer> &explorer) {
+                        auto buildFunction = [self](const CosmosLikeTransactionBuildRequest &request) {
                                 auto currency = self->getWallet()->getCurrency();
                                 auto tx = std::make_shared<CosmosLikeTransactionApi>();
-                                tx->setAccountNumber(self->getAccountUid());
                                 tx->setCurrency(self->getWallet()->getCurrency());
                                 tx->setFee(request.fee);
                                 tx->setGas(request.gas);
                                 tx->setMessages(request.messages);
                                 tx->setSigningPubKey(self->getKeychain()->getPublicKey());
-                                tx->setSequence(request.sequence);
```

Comment:
sequence isn't part of the tx anymore ?

Reply:
Nor in broadcast:
https://github.com/cosmos/cosmos-sdk/blob/2e42f9cb745aaa4c1a52ee730a969a5eaa938360/x/auth/types/stdtx.go#L23-L30
And neither in explorers, e.g:
https://hubble.figment.network/cosmos/chains/cosmoshub-3/blocks/951065/transactions/A65CF7D5634FF8D9442594A2AA9CED6C4CABA0D0CB4F5B99067A49A0D3EE5497?format=json

Reply:
Reverted the removal, as sequence is part of the signature payload format